src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.kt

```diff
@@ -529,64 +533,130 @@ class BookingResourceIntTest : ResourceTest() {
     assertThatJsonFileAndStatus(response, 200, "offender_extra_info.json")
   }
 
-  @Test
-  fun fullOffenderInformation_byOffenderNo() {
-    val response = testRestTemplate.exchange(
-      "/api/bookings/offenderNo/{offenderNo}?extraInfo=true",
-      GET,
-      createHttpEntity(NORMAL_USER, null),
-      String::class.java,
-      "A1234AG",
-    )
-    assertThatJsonFileAndStatus(response, 200, "offender_extra_info.json")
-  }
+  @Nested
+  @DisplayName("/api/bookings/offenderNo/{offenderNo}")
+  inner class OffenderDetails {
 
-  @Test
-  fun fullOffenderInformation_byOffenderNoAlt() {
-    val response = testRestTemplate.exchange(
-      "/api/offenders/{offenderNo}",
-      GET,
-      createHttpEntity(NORMAL_USER, null),
-      String::class.java,
-      "A1234AG",
-    )
-    assertThatJsonFileAndStatus(response, 200, "offender_extra_info.json")
-  }
+    @Test
```

Comment:
New tests added for authorisation/authentication - ensuring that the data from 4 removed endpoints is returned in the offenderDetail for all overrideRole values